service-worker.js

```diff
@@ -28,7 +28,7 @@
  * The value is defined in app.js and will be passed to Service Worker on initialization (to avoid duplication)
  * @type {String}
  */
-var CACHE_NAME;
+const CACHE_NAME;
```

Comment:
This can't be a constant, as this variable is assigned a value, later in this same file

Reply:
@mossroy You're right. I missed that when reviewing. Oddly it didn't throw an error when I tested SW mode. I guess browsers are not treating const strictly.